addons/mail/static/src/models/activity_mark_done_popover_content_view.js

```diff
@@ -1,11 +1,20 @@
 /** @odoo-module **/
 
+import { useRefToModel } from '@mail/component_hooks/use_ref_to_model';
 import { registerModel } from '@mail/model/model_core';
 import { attr, one } from '@mail/model/model_field';
 import { clear } from '@mail/model/model_field_command';
 
+import { onMounted } from '@odoo/owl';
```

Comment:
Is there a reason not to retrieve it from `window.owl` like we usually do? It is just a question and not a request for changes, I actually prefer this new way of doing things.